src/ex1/script.js

```diff
@@ -0,0 +1,158 @@
+const todoListElement = document.getElementById("todo-list")
+const addTodoButton = document.getElementById("add-todo-button")
+const todoInput = document.getElementById("todo-input")
+const clearAllTodosButton = document.getElementById("clear-all-todos-button")
+const sumTodos = document.getElementById("sum-todos")
+const orderSelect = document.getElementById("order-select")
+const enterTodos = document.getElementById("enter-todos")
```

Comment:
having all the elements outside here requires all the dom elements exist when this file is loaded, which might not be the case in complex web apps that dynamically change the dom depending on the view